NuGet.config

```diff
@@ -17,6 +17,8 @@
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vs-impl-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl-archived/nuget/v3/index.json" />
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <!-- DO NOT MERGE -->
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Do not approve until I delete this.

Reply:
Mirroring build https://dev.azure.com/dnceng/internal/_build/results?buildId=2179577&view=results